support-frontend/assets/components/checkoutBenefits/checkoutBenefitsList.tsx

```diff
@@ -72,34 +76,58 @@ const para = css`
 export type CheckoutBenefitsListProps = {
 	title: string;
 	checkListData: CheckListData[];
+	contributionType: ContributionType;
+	countryGroupId: CountryGroupId;
 	buttonCopy: string | null;
+	nudgeTitleTopCopy: string;
+	nudgeTitleCopy: string;
 	handleButtonClick: () => void;
+	onNudgeClick: () => void;
 };
 
 export function CheckoutBenefitsList({
 	title,
 	checkListData,
+	contributionType,
+	countryGroupId,
+	nudgeTitleTopCopy,
+	nudgeTitleCopy,
+	onNudgeClick,
 }: CheckoutBenefitsListProps): JSX.Element {
+	const [displayNudge, setDisplayNudge] = useState(true);
 	return (
 		<div css={container}>
-			<h2 css={heading}>{title}</h2>
-			<hr css={hr(`${space[4]}px 0`)} />
-			<table css={table}>
-				{checkListData.map((item) => (
-					<tr>
-						<td css={[checkListIcon, item.maybeGreyedOut]}>
-							<div css={iconContainer}>{item.icon}</div>
-						</td>
-						<td css={[checkListText, item.maybeGreyedOut]}>{item.text}</td>
-					</tr>
-				))}
-			</table>
+			{contributionType === 'ONE_OFF' && displayNudge && (
+				<CheckoutNudge
+					countryGroupId={countryGroupId}
+					nudgeTitleTopCopy={nudgeTitleTopCopy}
+					nudgeTitleCopy={nudgeTitleCopy}
+					onNudgeClose={() => setDisplayNudge(false)}
+					onNudgeClick={onNudgeClick}
+				/>
+			)}
+			{contributionType !== 'ONE_OFF' && (
+				<>
+					<h2 css={heading}>{title}</h2>
+					<hr css={hr(`${space[4]}px 0`)} />
+					<table css={table}>
+						{checkListData.map((item) => (
+							<tr>
+								<td css={[checkListIcon, item.maybeGreyedOut]}>
+									<div css={iconContainer}>{item.icon}</div>
+								</td>
+								<td css={[checkListText, item.maybeGreyedOut]}>{item.text}</td>
+							</tr>
+						))}
+					</table>
+					<hr css={hr(`${space[5]}px 0 ${space[4]}px`)} />
+					<p css={para}>Cancel anytime</p>
+				</>
+			)}
 			{/* <UpsellButton
 				buttonCopy={buttonCopy}
 				handleButtonClick={handleButtonClick}
 			/> */}
```

Comment:
We could delete this as the supporter plus nudge supersedes this old component

Reply:
Should be good to delete the component as the nudge component replaces the upsell button